Misc/NEWS

```diff
@@ -10,6 +10,9 @@ What's New in Python 3.7.0 alpha 1?
 Core and Builtins
 -----------------
 
+- bpo-30721: ``print`` now shows correct usage hint for using Python 2
+  redirection syntax. Patch by Sanyam Khurana.
+
```

Comment:
Blurb tool looks great! I faced a lot of merge conflicts earlier while rebasing my patches from the latest upstream changes.

This would greatly solve the problem.